UploadServlet.java

```diff
@@ -21,6 +21,11 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import javax.servlet.http.Part;
+import javax.swing.JOptionPane;
+
+import org.apache.catalina.core.ApplicationPart;
+
+import com.database.DB;

```

Comment:
@hema-001 

Line 34:  why use 1021 instead of 1024?

Hui

Reply:
I'm not sure I will have to ask @mjhmohamed .

It should be 1024 * 1024.

Thanks.